src/main/java/wolox/training/models/User.java

```diff
@@ -119,16 +128,26 @@ public void setBirthDate(LocalDate birthDate) {
         this.birthDate = birthDate;
     }
 
+    /**
+     * Set password of the user
+     *
+     * @param password
+     */
+    public void setPassword(String password) {
+        checkNotNull(password);
```

Comment:
Hash Password, gives more security to app. 
look this [link](https://www.baeldung.com/java-password-hashing)

Reply:
It is already hashed when arrives to model, it is already hashed by controller https://github.com/wolox-training/jd-java/blob/a0ef0bfd85d61fce67bf56c98edfa262be7f7eeb/src/main/java/wolox/training/controllers/UserController.java#L95 https://github.com/wolox-training/jd-java/blob/a0ef0bfd85d61fce67bf56c98edfa262be7f7eeb/src/main/java/wolox/training/controllers/UserController.java#L162